packages/operator/pkg/apiserver/routes/v1/training/model_training_validation.go

```diff
@@ -198,12 +196,12 @@ func (mtv *MtValidator) validateVCS(mt *training.ModelTraining) (err error) {
 		switch {
 		case vcs.Spec.Type != connection.GITType:
 			err = multierr.Append(err, fmt.Errorf(WrongVcsTypeErrorMessage, vcs.Spec.Type))
-		case len(vcs.Spec.Reference) == 0:
-			err = multierr.Append(err, fmt.Errorf(WrongVcsReferenceErrorMessage, vcs.ID))
-		default:
+		case len(vcs.Spec.Reference) != 0:
 			logMT.Info("VCS reference parameter is nil. Set the default value",
 				"name", mt.ID, "reference", vcs.Spec.Reference)
 			mt.Spec.Reference = vcs.Spec.Reference
+		default:
+			logMT.Info("Neither VCS connection or Training has reference specified, using default")
```

Comment:
```suggestion
			logMT.Info("Neither VCS connection or Training has reference specified, using VCS default branch")
```
to indicate that this branch is taken right from VC system.